modules/ROOT/examples/live-demos/inline/index.html

```diff
@@ -1,6 +1,6 @@
 <div class="demo-inline">
   <div class="container">
-    <p><img src="https://www.tiny.cloud/images/email-banner-2.png" /></p>
+    <p><img style="width: 100%;" src="{{imagesdir}}/tiny-banner.png" /></p>
```

Comment:
```suggestion
    <p><img src="{{imagesdir}}/tiny-banner.png" style="width: 100%;" /></p>
```
Just to keep this consistent with the 6.x docs.